crates/ra_ide/src/completion/completion_item.rs

```diff
@@ -285,6 +304,51 @@ impl Builder {
         self.deprecated = Some(deprecated);
         self
     }
+    #[allow(unused)]
+    pub(crate) fn compute_score(mut self, ctx: &CompletionContext) -> Builder {
```

Comment:
I think this function should live in `presentation.rs`, and here we only need `set_score`

Reply:
updated